src/use/use.py

```diff
@@ -177,11 +180,11 @@ def fail_or_default(default, exception, msg):
     else:
         raise exception(msg)
 
-def apply_aspect(mod:ModuleType, check:callable, pattern:str, decorator:callable):
+def apply_aspect(mod:ModuleType, check:callable, pattern:str, decorator:Callable[...]):
     """Apply the aspect as a side-effect, no copy is created."""
     # TODO: recursion?
     parent = mod
-    for name, obj in parent.__dict__.items():
+    for _, obj in parent.__dict__.items():
```

Comment:
why not .values() if we're on it?

Reply:
@amogorkon Good question, I dont have any memory of making this change! But I like your suggestion.

Reply:
Should be fixed